src/main/resources/application.yaml

```diff
@@ -144,6 +144,8 @@ deployer:
         enabled: true
         blacklist:
           # Indicates if the blacklist should be enabled for all targets
+          # (this will have no effect if the sandbox is disabled)
```

Comment:
Please clarify `sandbox`. Either name it clearly or link a doc. Ping me to discuss options.